mobile_net/utils/dataset.py

```diff
@@ -0,0 +1,86 @@
+# !/usr/bin/env python
+
+"""
+Loads all images into cache and create a tf.Dataset Object
```

Comment:
Shouldn't this be the docstring of the `Dataset` class (or rather its `_get_dataset` method)?